apps/prairielearn/src/server.js

```diff
@@ -95,6 +95,16 @@ if ('h' in argv || 'help' in argv) {
   process.exit(0);
 }
 
+function excludeRoutes(routes, handler) {
```

Comment:
I thought this was a bit kludgy, but ChatGPT agrees that this is the cleanest way to mount-on-all-paths-NOT-matching in Express, so I guess we have to accept it 😅

Reply:
Yeah, I think this is the best we can do. Hono has a similar helper function: https://hono.dev/docs/middleware/builtin/combine#except. Their implementation is more or less similar, though a bit more sophisticated:

https://github.com/honojs/hono/blob/7e17b76ce9438d68072a271bd593f7d5deb8542b/src/middleware/combine/index.ts#L130-L154

Reply:
Nice, `hono` is looking better all the time.